ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -586,6 +586,18 @@ static void setAutoTriggerEnabled(RaftProperties properties, boolean autoTrigger
       setBoolean(properties::setBoolean, AUTO_TRIGGER_ENABLED_KEY, autoTriggerEnabled);
     }
 
+    /** whether trigger snapshot when stop raft server */
+    String TRIGGER_SNAPSHOT_WHEN_STOP_ENABLED_KEY = PREFIX + ".trigger.snapshot.when.stop.enabled";
```

Comment:
Let's rename it to `PREFIX + ".trigger-when-stop.enabled"`.

Reply:
It took me a long time to come up with this name, and now thank you for the better name!